src/network/networkd-can.c

```diff
@@ -211,6 +211,12 @@ static int link_set_can(Link *link) {
                 log_link_debug(link, "%sabling triple-sampling", link->network->can_triple_sampling ? "En" : "Dis");
         }
 
+        if (link->network->can_berr_reporting >= 0) {
+                cm.mask |= CAN_CTRLMODE_BERR_REPORTING;
+                SET_FLAG(cm.flags, CAN_CTRLMODE_BERR_REPORTING, link->network->can_berr_reporting > 0);
+                log_link_debug(link, "%sabling bus error reporting", link->network->can_berr_reporting > 0 ? "En" : "Dis");
```

Comment:
Not a matter, but in this block `can_berr_reporting` can be treated as a boolean. So, it is ok to drop `> 0`.

Reply:
Oh, right - I could push an update or it could just be dropped while merging.

Reply:
i really dislike the weird way of putting together words btw.. (here and above in the pre-existing code, too) i.e. the "En"/"Dis" stuff. It sucks for translators, and is generally ugly. I mean, this is not going to be something we ever going to translate, but it's still ugly as fuck to concatenate english words in code like this. better would be to maybe do: `"Setting bus error reporting to '%s'.", enable_disable(…)=`...

Reply:
> Oh, right - I could push an update or it could just be dropped while merging.

Please use force-push.

And, if you like, please also address @poettring's comment. Or, I think it is ok to do that in another later PR.

Reply:
I do not like that either but just wanted to be consistent with the rest of the code. So I put a _pretty up_ commit in front and addressed yuwata's suggestion as well.

Reply:
Finally I did another _fly-by pretty up_ commit to also drop the `> 0` for `can_fd_mode` and `can_non_iso`.